cluster-controller/src/main/java/io/enmasse/barnabas/controller/cluster/resources/AbstractCluster.java

```diff
@@ -126,6 +158,33 @@ protected Volume createEmptyDirVolume(String name) {
         return volume;
     }
 
+    protected Volume createConfigMapVolume(String name, String configMapName) {
+
+        ConfigMapVolumeSource configMapVolumeSource = new ConfigMapVolumeSourceBuilder()
+                .withName(configMapName)
+                .build();
+
+        Volume volume = new VolumeBuilder()
+                .withName(name)
+                .withConfigMap(configMapVolumeSource)
+                .build();
+        log.info("Created configMap volume {}", volume);
+        return volume;
+    }
+
+    protected ConfigMap createConfigMap(String name, Map<String, String> data) {
+
+        ConfigMap metricsCm = new ConfigMapBuilder()
```

Comment:
Just a small detail ... but the function name is generic `createConfigMap`. So the config map should probably not be called `metricsCm` but meybe just `cm`.

Reply:
Right !